lib/base.ts

```diff
@@ -75,6 +75,10 @@ export class TranspilerBase {
 
   maybeVisitTypeArguments(n: {typeArguments?: ts.NodeArray<ts.TypeNode>}) {
     if (n.typeArguments) {
+      // If it's a single type argument `<void>`, ignore it and emit nothing.
```

Comment:
Nit: could you mention that this is for e.g. `Promise<void>`, and link to https://github.com/dart-lang/sdk/issues/2231#issuecomment-108313639 ?